src/base-cookie.js

```diff
@@ -23,6 +23,26 @@ const get = (name) => {
   }
 };
 
+const getAll = (name) => {
+  try {
+    const ca = document.cookie.split(';');
```

Comment:
nit: is this `ca` for `cookieArray` ?

Reply:
I suppose so. Its what's used in the regular `get` method above